programs/drift/src/math/amm_spread.rs

```diff
@@ -507,6 +513,7 @@ pub fn calculate_spread_reserves(
             .amm
             .quote_asset_reserve
             .safe_sub(quote_asset_reserve_delta.unsigned_abs())?
+            .max(1)
```

Comment:
dont need max/min in the other branch?

Reply:
i think the spread math works, you just need to bound for percentage spread on shorts